pkg/util/net.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2024 API Testing Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package util
+
+import (
+	"net"
+	"strings"
+)
+
+// GetPort returns the port of the listener
+func GetPort(listener net.Listener) string {
+	if listener == nil {
```

Comment:
Add a log info?

Reply:
This is a util method. I prefer to print the log from the caller. What do you think?